Source/WebCore/platform/ProcessIdentity.cpp

```diff
@@ -62,4 +62,13 @@ ProcessIdentity::ProcessIdentity(MachSendRight&& taskIdToken)
 }
 #endif
 
+ProcessIdentity ProcessIdentity::clone() const
```

Comment:
This addition proves a bit the shortcoming of this approach:
The language has this feature exactly to avoid this sort of redundant boiler plate code. Now we are spending extra work going against that.
The language has this feature exactly so nobody needs to come up with bespoke conventions of copy,duplicate,clone that then can get incoherent..

Reply:
It's true, but the language-based copy is pretty easy to trigger accidentally.

There's plenty of precedent in the codebase for adding an explicit `clone` function for types where we want it to be very explicit. that a copy will happen.